### ISSUE TITLE
[UX] Make the sending of an ENML note easier

### ISSUE THREAD

**Issue by Taluu** [evernote/evernote-cloud-sdk-php#7]
when using, as the examples, the `PlainTextNoteContent` model to post a note content, we just have to use this class :

```php
<?php

// ...

$note = new Note;
$note->content = new PlainTextNoteContent($content);
```

But if you need to use the `EnmlNoteContent` instead, you need to decorate it with some XML first :

```php
<?php

// ...

$content = <<<CONTENT
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE en-note SYSTEM "http://xml.evernote.com/pub/enml2.dtd">
<en-note>$content</en-note>
CONTENT;

$note = new Note;
$note->content = new EnmlNoteContent($content);
```

I think something is wrong on the usage here, and that the node should be able to decorate the text itself, in order to do that :

```php
<?php

// ...

$note = new Note;
$note->content = new EnmlNoteContent($content);
```

Especially as this is not documented, this can be a headache if you don't know the little tip...

**Comment by Laurent-Sarrazin** [comment]
Hi Baptiste, 

Thanks for the feedback. I had this idea at one point and forgot about it. 
Will certainly improve this part soon (and at least document it).

Laurent

**Comment by Laurent-Sarrazin** [comment]
Fixed by 603e97839c2741de4eea74a39f99586032ab1660
